src/common/file-url-provider.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { provideBlob } from './blob-provider';
+import { WindowUtils } from './window-utils';
+
+export class FileURLProvider {
+    constructor(private windowUtils: WindowUtils) {}
+
+    public provideURL = (blobParts?: any[], mimeType?: string): string => {
```

Comment:
Will this cause error if both params are not provided? i.e. calling provideBlob()

Reply:
This is a good idea for one of the test cases that @annikakouhia can add for this file.

Reply:
Also I think a test case should be added in blob provider.

Reply:
Made them both mandatory parameters--blobs can be created without both, but for our purposes we want both values